src/StronglyTypedId.Attributes/StronglyTypedIdJsonConverter.cs

```diff
@@ -6,6 +6,6 @@
 [Flags]
 public enum StronglyTypedIdJsonConverter
 {
-    NewtonsoftJson = 1,
-    SystemTextJson = 2
+    NewtonsoftJson = 0,
+    SystemTextJson = 1
```

Comment:
This should be reverted: implementation uses `HasFlag` method, so numeration must follow 1,2,4,8,....

Reply:
Great point, thanks! Reverted it now :)